addons/im_livechat/views/im_livechat_channel_views.xml

```diff
@@ -83,7 +81,7 @@
                             <button class="oe_stat_button" type="action" attrs="{'invisible':[('nbr_channel','=', 0)]}" name="%(mail_channel_action_from_livechat_channel)d" icon="fa-comments">
                                 <field string="Sessions" name="nbr_channel" widget="statinfo"/>
                             </button>
-                            <button name="action_view_rating" attrs="{'invisible':[('rating_percentage_satisfaction','=', -1)]}" class="oe_stat_button" type="object" icon="fa-smile-o">
+                            <button name="action_view_rating" attrs="{'invisible':[('rating_percentage_satisfaction', '&lt;=', 0)]}" class="oe_stat_button" type="object" icon="fa-smile-o">
```

Comment:
This condition is not correct. Having a rating of 0 is not the same as having no rating.